ARCs/arc-0026.md

```diff
@@ -56,10 +70,25 @@ The scheme component ("algorand:") is case-insensitive, and implementations must
 
 - amount: microAlgos or smallest unit of asset 
 
-- asset: The asset id this request refers to (if Algos, simply omit this parameter) 
+- asset: The asset id this request refers to (if Algos, simply omit this parameter). When the method is available, the asset is an Asset Array member for the ABI call.
+
+- method: The full ABI method to call. E.g. claim(uint64,uint64)byte[]
+
+- arg: The argument to be added to ABI call arguments array.
+
+- box: The box name reference.
+
+- fee: Optional static transaction fee.
```

Comment:
I think it is worth mentioning that this number MUST be expressed in microALGO.

Reply:
Done! Committed! Thanks for the great points again. I'm grateful @cusma !